docs/tutorials/get-started/make-an-led-blink-with-the-viam-app.md

```diff
@@ -101,7 +101,7 @@ The long anode should be connected to the resistor and the short cathode should
 ## Configuring your bot using the Viam app
 
 Before proceeding, be sure that you have connected your Pi to the Viam app.
-Steps on how to do this can be found in the [Adding your Pi on the Viam App](/installation/install/linux-install/) section of the Viam Documentation.
+Steps on how to do this can be found in the [Adding your Pi on the Viam App](/installation#install-viam-server) section of the Viam Documentation.
```

Comment:
```suggestion
Steps on how to do this can be found in the [adding your Pi on the Viam app](/installation#install-viam-server) section of the Viam Documentation.
```
Opt. to match above